spec/css/moz_document/functions/interpolated/warning

```diff
@@ -5,9 +5,7 @@ relase. For details, see http://bit.ly/moz-document.
   ,
 1 | / @-moz-document url(#{"sass-lang.com"}) {
 2 | |   a {type: unquoted full url}
-3 | | }
-4 | | @-moz-document url(#{sa + ss}-lang.com) {
-  | '^
+3 | \ }
   '
 
```

Comment:
Are you expecting to no longer have a carrot pointing to the error position for all of these?

Reply:
Yep. It now indicates that the entire first `@-moz-document`, and only that one, is highlighted.